portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeAccountNav.vm

```diff
@@ -85,9 +85,12 @@ For opening help sidebar
           aria-controls="sakai-notificationsPanel"
           aria-label="$rloader.sit_notifications"
           title="$rloader.sit_notifications">
-        <i class="bi-bell"></i>
-        <span class="portal-notifications-indicator p-1 bg-danger rounded-circle" style="display: none">
-          <span class="visually-hidden">There are new notifications available</span>
+        <i class="portal-notifications-icon bi-bell"></i>
+        <i class="portal-notifications-no-permissions-indicator si si-warning d-none"
+           aria-label="$rloader.sit_notifications_not_permitted_label">
```

Comment:
aria-hidden?

Reply:
don't we want that aria-label read out by screen readers?

Reply:
Or does it not work like that? Will the screen reader read the tag and the label out? Unless we use aria-hidden as well?

Reply:
you're exactly correct. sorry